### ISSUE TITLE
Wrong expression on EXPLAIN Sort Key with Custom Aggregates

### ISSUE THREAD

**Issue by aamederen** [citusdata/citus#1571]
Notice the `Sort Key` in the following explain output is confusing and wrong:

```
postgres=# \d hll_explain
 Table "public.hll_explain"
 Column  | Type | Modifiers
---------+------+-----------
 partcol | text |
 hllcol  | hll  |

postgres=# explain select hll_cardinality(sum(hllcol)) o from hll_explain order by o;

                                                   QUERY PLAN
----------------------------------------------------------------------------------------------------------------
 Sort  (cost=0.00..0.00 rows=0 width=0)
   Sort Key: hll_cardinality(sum((hll_cardinality(sum(remote_scan.o)))))
   ->  Aggregate  (cost=0.00..0.00 rows=0 width=0)
         ->  Custom Scan (Citus Real-Time)  (cost=0.00..0.00 rows=0 width=0)
               Task Count: 32
               Tasks Shown: One of 32
               ->  Task
                     Node: host=localhost port=3700 dbname=postgres
                     ->  Aggregate  (cost=21.00..21.01 rows=1 width=32)
                           ->  Seq Scan on hll_explain_102072 hll_explain  (cost=0.00..18.80 rows=880 width=32)
(10 rows)
```

```
remote_scan.o -> probably hll
sum(remote_scan.o) -> returns hll
hll_cardinality(sum(remote_scan.o)) -> returns number
sum((hll_cardinality(sum(remote_scan.o))) -> returns number
hll_cardinality(sum((hll_cardinality(sum(remote_scan.o))))) -> invalid
```

**Comment by metdos** [comment]
@aamederen, but we are running correct queries on the workers and master, right?

**Comment by aamederen** [comment]
@metdos yes, the query runs correctly

**Comment by anarazel** [comment]
That's not a 7.0 bug, right?

**Comment by marcocitus** [comment]
Fixed by #3098
